akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FutureDirectivesSpec.scala

```diff
@@ -19,6 +24,12 @@ class FutureDirectivesSpec extends RoutingSpec {
     case e: TestException ⇒ complete((StatusCodes.InternalServerError, "Oops. " + e))
   }
 
+  trait TestWithCircuitBreaker {
+    val breakerResetTimeout = 500.millis
+    val breaker = new CircuitBreaker(system.scheduler, maxFailures = 1, callTimeout = 10.seconds, breakerResetTimeout)
+    def openBreaker = breaker.withCircuitBreaker(Future.failed(new Exception("boom")))
```

Comment:
`openBreaker()` with empty parens here and at the call sites to signal side effect?